app/models/signal.py

```
@@ -0,0 +1,213 @@
+"""Signal intelligence models supporting enterprise delivery."""
+
+import uuid
+import uuid
```

Ruff (0.13.3)

Redefinition of unused `uuid` from line 3

Remove definition: `uuid`

(F811)